pkg/apis/kubeone/v1beta2/types.go

```diff
@@ -164,6 +164,9 @@ type HostConfig struct {
 	// PublicAddress is externally accessible IP address from public internet.
 	PublicAddress string `json:"publicAddress"`
 
+	// IPv6AddressList
```

Comment:
maybe add some description as well?

Reply:
well `list of IPv6 addresses` doesn't say much more :wink: 

will add

Reply:
What about something like `List of IPv6 addresses of the node, only the first one will be announced to the k8s control plane`?
Or in other words, explaining why this is a list and that the first one is important from k8s perspective.

Reply:
Makes sense. Stealing...